ci-operator/step-registry/upi/deprovision/heterogeneous/vpc/upi-deprovision-heterogeneous-vpc-commands.sh

```diff
@@ -59,38 +54,15 @@ function retry {
   done
 }
 
-function setup_jq() {
-  if [ -z "$(command -v jq)" ]
-  then
-    echo "jq is not installed, proceed to installing jq"
-    curl -L "https://github.com/jqlang/jq/releases/download/jq-${JQ_VERSION}/jq-linux64" -o /tmp/jq && chmod +x /tmp/jq
-  fi
-}
-
-function setup_ibmcloud_cli() {
-  if [ -z "$(command -v ibmcloud)" ]
-  then
-    echo "ibmcloud CLI doesn't exist, installing"
-    curl -fsSL https://clis.cloud.ibm.com/install/linux | sh
-  fi
-
-  retry "ic config --check-version=false"
-  retry "ic version"
-}
-
 function setup_terraform_cli() {
-  if [ -z "$(command -v terraform)" ]
-  then
-    echo "terraform CLI doesn't exist, installing"
-  fi
   curl -L "https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip" -o "${IBMCLOUD_HOME_FOLDER}"/terraform.zip
   cd "${IBMCLOUD_HOME_FOLDER}" || true
   unzip -o "${IBMCLOUD_HOME_FOLDER}"/terraform.zip
   ${IBMCLOUD_HOME_FOLDER}/terraform version
 }
 
 function cleanup_ibmcloud_vpc() {
-  cos_name="${NAME_PREFIX}-mac-intel-cos"
+  cos_name="${NAME_PREFIX}-multi-arch-intel-cos"
```

Comment:
```suggestion
  cos_name="${NAME_PREFIX}-multi-arch-intel-cos"
```
We'd have to update our automation to use -multi-arch- https://github.com/IBM/ocp4-upi-compute-powervs-ibmcloud/blob/ba3a92b156fc3c3c93d60e6aca761e6621e4bb6d/modules/5_image/image.tf#L18

That should be a quick update. If you want to create an issue on our internal openshiftp jira project, we can do it quickly.

Reply:
Sure , I ad created the PR - https://github.com/IBM/ocp4-upi-compute-powervs-ibmcloud/pull/21 
As of now jira@ibm is down, once it up , will open an issue and will update the PR details there .